doneill/er-cli-go
# Init Viper write cofig

Persist the results of the auth command for future use in other commands

**Manual Test:**
- Build and run the app
- Confirm token results no longer display in terminal
- Confirm **er.toml** file generated with `key:values` representing the following:
  - `username`
  - `oauth_token`
  - `expires`
